nnpdf_data/nnpdf_data/theory.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain. 
+It also implement some utilities function to manage the theory cards. 
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class TheoryCard:
+    ID: int
+    PTO: int
+    FNS: str
+    DAMP: int
+    DAMPPOWERc: None #Power of the damping factor in FONLL for the c
+    DAMPPOWERb: None #Power of the damping factor in FONLL for the b
+    IC: int
+    ModEv: str
```

Comment:
Agreed

Reply:
`IterEv` shouldn't be placed in the operator card?

Reply:
As `ModEv` (`evolution_method` in EKO) and `ModSV` (`scvar_method` in EKO). They are all part of the operator card.

But I have the feeling that they are ignoring the distinction, and flattening everything...

Reply:
Actually the idea was to keep here all the parameters that can actually distinguish a theory from another one, so if we have different theories differing only for `IterEv`, let's keep it here, although it's not fully consistent for instance with `polarized`.

Reply:
Maybe we should bring here polarized as well.

Reply:
Our idea for keeping it in the operator card was that at some point we'd like to have mixed pol, unpol theories and observables (as the g1/F1 ratio), same holds for timelike.

Reply:
The information on whether it is a mixed theory is also relevant. Note that these should be the same theory card for pineko / pinefarm and nnpdf.
The idea is that pineko / pinefarm (and nnpdf) use the information here to pass through to the interested parties what they need (so here it can be called `polarized: mixed/always/never` for instance, doesn't need to match -and shouldn't- the parameter in eko)

Reply:
Indeed, even if a parameter belongs to the operator card, we should also keep it here and then `pineko` will take care of reading it from the theory card and writing it on the operator card. Otherwise we will keep having the same problem, that some of the parameters are hidden in a file which has no standards and which is not stored in NNPDF.

Reply:
I see your point, then polarized really depends on the observables, so I'd not add it here.